hoppfish/hwangsae-recorder-user-command.py

```diff
@@ -31,6 +34,16 @@ def get_json_value(obj, key, default):
 
     return ret
 
+def internal_error():
+    message = {
+            'status': 404,
+            'message': 'internal error :',
+    }
+    resp = jsonify(message)
+    resp.status_code = 404
+
+    return resp
```

Comment:
I guess this could be simpler. How about this?

```
def make_internal_error(error_code, error_message):
  return jsonify(status=error_code, message=error_message)
```